packages/react-data-grid/src/common/types.ts

```diff
@@ -254,6 +254,10 @@ export interface ScrollPosition {
   scrollTop: number;
 }
 
+export interface ScrollOption extends ScrollPosition {
+  reachedBottom?: boolean;
```

Comment:
We might want to revisit the API name, but I believe this should be a reasonable API.

Reply:
Or should I add a stand alone API like `onReachedBottom` to be explicitly?